From: Matthew Wilcox <matthew.r.wilcox@intel.com>
Date: Fri, 6 May 2011 08:37:54 -0400
Subject: NVMe: Add include of delay.h for msleep
Git-commit: fd63e9ceeeae58cfe877c2d49d41c1bf7532303c
References: FATE#313627
Patch-Mainline: v3.7-rc1

Previously it was being implicitly included through some other header file

Signed-off-by: Matthew Wilcox <matthew.r.wilcox@intel.com>
Acked-by: Hannes Reinecke <hare@suse.de>
---
 drivers/block/nvme.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/block/nvme.c b/drivers/block/nvme.c
index 9c0ab2a..b285a7e 100644
--- a/drivers/block/nvme.c
+++ b/drivers/block/nvme.c
@@ -20,6 +20,7 @@
 #include <linux/bio.h>
 #include <linux/bitops.h>
 #include <linux/blkdev.h>
+#include <linux/delay.h>
 #include <linux/errno.h>
 #include <linux/fs.h>
 #include <linux/genhd.h>
-- 
1.7.4.2

